packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -87,6 +90,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const typingIndicatorProps = usePropsFor(TypingIndicator);
   const headerProps = useAdaptedSelector(getHeaderProps);
   const errorBarProps = usePropsFor(ErrorBar);
+  const fileUploadButtonProps = useSelector(fileUploadButtonSelector);
```

Comment:
mark as beta here

Reply:
`ChatScreen` is a fairly large component. I'd recommend avoiding any conditional compilation in this component.

Instead, conditionally compile inside `FileUploadButton` to return `<></>` on stable flavor builds. It'll likely end up being cleaner.

Reply:
Also, shouldn't this be `useAdapterSelector`?

Reply:
This selector is only return userId and not anything related to the StatefulChatClient State.
useSelector should be fine here.
Correct me if I am wrong @PorterNan ?

Reply:
@prprabhu-ms Can I incorporate if/else in conditional compilation?
I don't fully understand how I can do what you are suggesting.
Is this a good example
![image](https://user-images.githubusercontent.com/9782747/151071102-3ac67b92-cfee-4811-bfb2-faaedd1c0ae1.png)